generatedocs/data/actions/publishbookmark/examples.md

```diff
@@ -0,0 +1,27 @@
+### Example
+
+Publish bookmark previously created in the script with an id `bookmark1`.
+
+```json
+{
+    "label" : "Publish bookmark 1",
+    "action": "publishbookmark",
+    "disabled" : false,
```

Comment:
Is this parameter described?

Reply:
disabled? yes:
Structure of an action entry

All actions follow the same basic structure:

    action: Name of the action to execute.
    label: (optional) Custom string set by the user. This can be used to distinguish the action from other actions of the same type when analyzing the test results.
    disabled: (optional) Disable action (true / false). If set to true, the action is not executed.
    settings: Most, but not all, actions have a settings section with action-specific settings.